-------------------------------------------------------------------------------
Test set: com.alaythiaproductions.sfgpetclinic.SfgPetClinicApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.886 s <<< FAILURE! - in com.alaythiaproductions.sfgpetclinic.SfgPetClinicApplicationTests
contextLoads  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.alaythiaproductions.sfgpetclinic.model.Visit.pet references an unknown entity: com.alaythiaproductions.sfgpetclinic.model.Pet
Caused by: org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.alaythiaproductions.sfgpetclinic.model.Visit.pet references an unknown entity: com.alaythiaproductions.sfgpetclinic.model.Pet

